### ISSUE TITLE
[question]: fetch from upstream interval 

### ISSUE THREAD

**Issue by daviddean-x** [torbenraab/plane#21]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Dear @torbenraab 
I which interval will you fetch updates from upstream?
Sometimes there are useful commits and fixes, hope your fine,
best Regards,
Dan

### Why should this be worked on?

Notice

**Comment by torbenraab** [comment]
I try and fetch updates form the upstream as soon as there is a new release. I mostly try it that the release here is only 48 hours later online as the official release. Unfortunately I was ill for a while now (currently recovering) and had not have enough time to do the fetching. I think I can spare some time today.
If there is any idea on your side to do the fetching automatically I'm open to it.

**Comment by torbenraab** [comment]
Just put the release v0.16-dev online

**Comment by daviddean-x** [comment]
Thank you,
Do you have discord or other channels to discuss?

**Comment by torbenraab** [comment]
You can contact me on Discord via trbnrb or via Matrix @toraab:lila.systems

**Comment by daviddean-x** [comment]
@torbenraab Hey you, will update your repo next time? 
I had see plane is working on oidc implementation for a one time payment. In your PR they told OIDC will be free for people who was also flowering your PR? 
Hope your fine,
Best Regards

**Comment by torbenraab** [comment]
Hey @thefiredragon, I currently have a heavy workload so I will skip the 0.19. The next version will be up after the next update.
I was not informed before the email that got to everybody subscribed about the one time payment. As there is a payment this repo will be maintained further.
Hope this helps

**Comment by daviddean-x** [comment]
I think I had misunderstood something in the pr thread. But thank you :)

**Comment by torbenraab** [comment]
I don't think you misunderstood. I got from the pr that they might contact me according the pricing but this never happened. I thought about the one time payment but as the Gitlab integration is not ready and not even in sight I won't buy it for my company.

**Comment by daviddean-x** [comment]
They also want licencing each workspace for plane one... 
And will workspaces split into teams.

Gitlab would also be important, yesterday I had wrote over discord with some people from plane.

**Comment by daviddean-x** [comment]
@torbenraab I have a little question, we're still running your fork and plane has released plane one. 

Will you maintain this fork in future? If not we need to look if it's possible to upgrade to plane one with SSO without loosing our data. A short response here would be helpful,
 Best regards 
Dan

**Comment by torbenraab** [comment]
@thefiredragon Yes already working on it. As the authentication got reworked I have to look a lot deeper into it as I liked. I really hope they don't keep changing this as much as they did recently

**Comment by daviddean-x** [comment]
Thank you for the fast answer :) hope the best 😊

**Comment by torbenraab** [comment]
I'm sorry that there wasn't an update until now. I'm currently working on the newly released v0.22-dev version from plane. There have been a lot of changes in the code according the authentication. I took me some time to get what was done there. I hope I can get it working in the next days.

**Comment by torbenraab** [comment]
Current work is done in #31

**Comment by daviddean-x** [comment]
Thank you
I will test it l today in the evening, is there something to know before I try to update?
Best regards

**Comment by torbenraab** [comment]
There are some points that aren't implemented yet.
- Autodiscover
- Logout using the EndSession URL
I want to take a deeper look into those topics if it is asked.
But this implementation should be working better using a backend generated state

**Comment by torbenraab** [comment]
@thefiredragon Thanks for testing! I'm currently on vacation and have quite bad internet reception here. So a build takes forever!

**Comment by kotasudhakar** [comment]
Hi @torbenraab could you please update this fork with the upstram as this is missing a lot of features 
Would be really helpful if you can do so, or atleast guide me on the same and i can raise a PR for the same.

**Comment by torbenraab** [comment]
#32 Currently updating to v0.22 and pulling for preview soon

**Comment by torbenraab** [comment]
AIO Image will not be made available as the Github Hosted Actions Agent doesn't have suffiened resources

**Comment by kotasudhakar** [comment]
> AIO Image will not be made available as the Github Hosted Actions Agent doesn't have suffiened resources

If I understand it correctly, the images won't be hosted but the docker image can be built by ourselves right?

**Comment by torbenraab** [comment]
All other images will be hosted, but not the AIO for now. I will look into it.
You can build everything yourself at any time.

**Comment by kotasudhakar** [comment]
> All other images will be hosted, but not the AIO for now. I will look into it. You can build everything yourself at any time.


Got it, and what does AIO stands for?

Also, just wondering as i just saw that you made a release of 0.22 dev, but for some reason i can still see this is 419 commits behind the upstream. Please let me know if i am missing something here


![image](https://github.com/user-attachments/assets/6f9f0bac-f13d-4780-b409-92ae7bcfad5f)

**Comment by torbenraab** [comment]
There was no release since 2 month in the official repo. I will pull the preview tomorrow and see what i can do, but it will only be available via the preview tag as there was no official release 

AIO is the all in one image, but as we are not using it in the company I won’t have time or budget to maintain it.

**Comment by kotasudhakar** [comment]
> There was no release since 2 month in the official repo. I will pull the preview tomorrow and see what i can do, but it will only be available via the preview tag as there was no official release
> 
> AIO is the all in one image, but as we are not using it in the company I won’t have time or budget to maintain it.

Sorry to bother you again, by any chance did you got a chance to look at it?

**Comment by torbenraab** [comment]
@kotasudhakar I'm sorry but my job and paid projects come first. I will look into it as soon as possible.

**Comment by kotasudhakar** [comment]
> @kotasudhakar I'm sorry but my job and paid projects come first. I will look into it as soon as possible.

Sure, No problem.

**Comment by kotasudhakar** [comment]
hi, new version of plane has been released. can you please release the latest tag of the original ones?

Also, can you please tell me  how are you doing it? as in general for these i just rebase with the remote origin branch, however in here it is a little different and i can see something called as merge commits
